psydac/api/ast/utilities.py

```diff
@@ -347,7 +369,7 @@ def compute_atoms_expr_mapping(atomic_exprs, indices_quad,
         val_name = _print(atom) + '_values'
         val      = IndexedBase(val_name)[indices_quad]
         updates += [AugAssign(val,'+',Mul(*args))]
-    # ...
+        # ...
```

Comment:
@saidctb Again, I think it is appropriate to add a docstring also to this function.

Reply:
done !